SwiftWisdom/Core/StandardLibrary/Numbers/CountableClosedRange+Extensions.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CountableClosedRange+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Andrew Dolce on 11/21/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+public extension CountableClosedRange where Bound: SignedInteger {
```

Comment:
It hasn't been tested with different strides. I will address that before merging.